CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,157 @@
+  - Authoring or reviewing PRs on GitHub. At least one PR must be **merged**.
+  - Filing or commenting on issues on GitHub
+  - Contributing to a project, or community discussions (e.g. meetings, Slack,
+    email discussion forums, Stack Overflow)
+- Active contributor to Zarf
+
+#### Promotion process `NEEDS REFINED?`
+
+- Make at least one commit to a repository's code or open a pull request that gets merged into the repository
+
+### Reviewers
+
+Reviewers are trusted members of the Zarf community that are able to review changes to Zarf and indicate if those changes are ready for merge.  Reviewers have a strong and active track record of contribution to the Zarf project.
+
+#### Responsibilities
+
+Commits to being an active contributor and reviewer as part of the Zarf project.  'Active' is defined as six PRs or PR reviews (or mix thereof) in six months.
+Is supportive of new and occasional contributors and helps get useful PRs in shape to commit.
+
+#### Additional Privileges
+
+Has GitHub or CI/CD rights to approve pull requests in specific directories
+
+#### Prerequisites
+
+Experience as a Contributor for at least 6 months
+Is an Organization Member
+Has reviewed, or helped review, at least 10 (`?`)Pull Requests
+Has analyzed and resolved test failures
+Has demonstrated an in-depth knowledge of Zarf
+
+#### Promotion process
+
+- Sponsored by a maintainer
+  - With no objections from other maintainers
+  - Done through PR to update the CODEOWNERS file, and addition to Zarf Maintainer group
+- May self-nominate or be nominated by a maintainer
+  - In case of self-nomination, sponsor must comment approval on the PR
+
+### Maintainers
+
+Maintainers are responsible for the project's overall health.
+They are the only ones who can merge code contributions.
+While code review is focused on code quality and correctness,
+approval is focused on holistic acceptance of a contribution including:
+
+- Backwards/forwards compatibility
+- Adherence to API and style conventions
+- Subtle performance and correctness issues
+- Interactions with other parts of the system
+- Consistency between code and documentation
+
+#### Responsibilities
+
+- Demonstrate sound technical judgment
+- Maintain project quality control via code reviews
+  - Focus on holistic acceptance of contribution
+- Be responsive to review requests
+- Mentor contributors and reviewers
+- Approve and merge code contributions as appropriate
+- Participate in OpenSSF or Zarf-specific community meetings, if possible
+- Facilitating Zarf-specific community meetings, if possible
+
+#### Additional Privileges
+
+- Maintainer status may be a precondition to accepting especially large code contributions
+
+#### Pre-requisites
+
+- Reviewer for at least 1 month
+- Reviewed at least 10 substantial PRs to the codebase
+- Reviewed or got at least 30 PRs merged to the codebase
+
+```or```
+
+- Be a member of the Defense Unicorns or Radius Method organizations
```

Comment:
I don't think I inherently agree with this - but it is a little sticky. How do we codify this appropriately?